assets/router/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: openshift-cluster-ingress-router
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
Lack of this annotation was the reason my DaemonSet (which uses a node selector) wasn't scaling up after the isolated namespace refactor... lesson learned

Reply:
I thought you were setting this explicitly in the Go code before.  Did that not work for some reason, or do you mean the Go code that did this got lost in the isolated namespace refactor?

Reply:
Previously I was testing everything in the `default` namespace, which already had the annotation